components/quote-carousel/QuoteCarousel.style.js

```diff
@@ -41,7 +41,6 @@ const ContentWrapper = styled(motion.div)`
 const TextWrapper = styled.div`
```

Comment:
<img width="492" alt="Screen Shot 2021-05-12 at 12 50 01 PM" src="https://user-images.githubusercontent.com/33647880/118013871-94b2c000-b320-11eb-891b-52ad9d6eada3.png">
did the image rendering get messed up

Reply:
fixed in another PR